docs/learn/learn-governance.md

```diff
@@ -437,6 +483,62 @@ Fast-tracked referenda are the only type of referenda that can be active alongsi
+
+## Polkadot Fellowship
+
+The Fellowship is a mostly self-governing expert body with a primary goal of representing humans who embody and contain the technical knowledge base of the Polkadot network and protocol. This is accomplished by associating a rank with members to categorize the degree to which the system expects their opinion to be well-informed, of a sound technical basis and in line with the interests of Polkadot.
+
+Unlike the current Technical Collective it is designed to be far broader in membership (i.e. to work well with even tens of thousands of members) and with far lower in barriers to entry (both in terms of administrative process flow and expectations of expertise). Becoming a candidate member in the Fellowship is as easy as placing a small deposit.
+
+Members of the Fellowship can vote on any given Fellowship proposal and the aggregate opinion of the members (weighted by their rank) constitutes the Fellowship's considered opinion.
+
+The mechanism by which the Fellowship votes is the same (Substrate pallet) as what is used for Polkadot stakeholder voting for a proposed referendum.
+
+### Ranking System
+
+So how exactly does this ranking system work?
+
+To prevent a small group of participants from gaining effective control over the network this system will adhere to three main principles:
+
+1. The Fellowship must never have hard power over the network: it cannot change the parameters, conduct rescues or move assets. Their only power in governance resides in the the ability to reduce the effective timeline on which a referendum takes place.
+2. The Fellowship weights those with a higher rank more in the aggregate opinion, however the weight should not be so high as to make a small number of higher members’ opinions be insurmountable when compared to a coherent opinion coming from lower-ranked membership.
+3. The Fellowship should be designed to grow and develop its membership and their aggregate levels of expertise and in doing so ensure that its overall decision-making capacity gets stronger over time.
+
+To support these conditions, the Fellowship will have a constitution which outlines the requirements and expectations for individuals to attain and retain any given rank. Higher ranks are able to vote and promote lower ranks based on this constitution.
+
+**Demotion** occurs automatically after a given period has elapsed and the member is unable to defend their position to their peers.
+
+**Suspension** can happen only through general referendum, ensuring controversy or unpopularity with the Fellowship alone does not necessarily result in expulsion.
+
+Gaining access to top tiers of the ranks will also require a referendum, as to prevent the Fellowship from becoming a cabal (popularity with Fellowship peers alone is not enough to gain access to a top rank).
```

Comment:
```suggestion
To prevent the Fellowship from becoming a cabal (popularity with Fellowship peers alone should not be enough to gain access to a top rank), gaining access to top tiers of the ranks will require a referendum. 
```